.circleci/config.yml

```diff
@@ -45,3 +45,64 @@ jobs:
     - store_artifacts:
         path: ~/project/out
         destination: crc
+
+  build_docs:
+    machine:
+      image: ubuntu-1604:201903-01
+    steps:
+    - checkout
+    - run:
+        name: Update repo
+        command: sudo apt -y update
+    - run:
+        name: Add podman ppa
+        command: sudo add-apt-repository -y ppa:projectatomic/ppa && sudo apt -y update
+    - run:
+        name: Update repo
+        command: sudo apt -y update
+    - run:
+        name: Install podman
+        command: sudo apt -y install podman
+    - run:
+        name: Check podman
+        command: podman info
+    - run:
+        name: Make build_docs
+        command: make build_docs
+    - persist_to_workspace:
+        root: docs/
+        paths: build
+
+  docs_deploy:
+    docker:
+      - image: node:8.10.0
+    steps:
+    - checkout
+    - attach_workspace:
+        at: docs/build
+    - add_ssh_keys:
+        fingerprints:
+          - "c5:64:d4:79:8f:c7:9b:80:b8:16:71:20:57:6b:8c:94"
+    - run:
+        name: Install and configure dependencies
+        command: |
+          npm install -g --silent gh-pages@2.0.1
+          git config user.email "codereadycontainers@gmail.com"
+          git config user.name "crc-bot"
+    - run:
+        name: Deploy docs to gh-pages branch
+        command: gh-pages -d docs/build/* -s *.html --message "[skip ci] Docs update"
```

Comment:
This commit message is needed to stop circle ci running the job again, but since we don't have a circle ci config in the gh-pages branch, if ran it'll always fail.

Reply:
@anjannath I am not able to see where are we pushing the resulted html to `gh-pages` branch does that automatic pushed there or we have to specify it someplace?

Reply:
@praveenkumar Its pushed in line 94, the `gh-pages` is a nodejs tool, which is doing it

Reply:
> `npm install -g --silent gh-pages@2.0.1`

I do not see WHY we need this. it is just a `git` action to push to an orphaned branch.

Reply:
Remove the dependency, using just `git` now